README.md

```diff
@@ -62,35 +62,42 @@ Select *Stash Pull Request Builder* then configure:
 
 ## Building the merge of Source Branch into Target Branch
 
-You may want Jenkins to build the merged PR (that is the merge of `sourceBranch` into `targetBranch`) to catch any issues resulting from this. To do this change the Branch Specifier from `origin/pr/${pullRequestId}/from` to `origin/pr/${pullRequestId}/merge`
+### Letting Stash do the merge
 
-If you are building the merged PR you probably want Jenkins to do a new build when the target branch changes. There is an advanced option in the build trigger, "Rebuild if destination branch changes?" which enables this.
+You may want Jenkins to build the merged PR (that is the merge of `sourceBranch` into `targetBranch`) to catch any issues resulting from this. To do this, change the Branch Specifier from `origin/pr/${pullRequestId}/from` to `origin/pr/${pullRequestId}/merge`
+
+If you are building the merged PR, you probably want Jenkins to do a new build when the target branch changes. There is an advanced option in the build trigger, "Rebuild if destination branch changes" which enables this.
 
 You probably also only want to build if the PR was mergeable and always without conflicts. There are advanced options in the build trigger for both of these. 
 
-**NOTE: *Always enable `Build only if Stash reports no conflicts` if using the merge RefSpec!*** This will make sure the lazy merge on stash has happened before the build is triggered.
+**NOTE: *Always enable `Build only if Stash reports no conflicts` if using the merge RefSpec!*** This will make sure the lazy merge on Stash has happened before the build is triggered.
+
+### Merging locally, variant 1
```

Comment:
I'm not sure if this is useful, maybe if would be better to just leave a single version using the `../from` refspec? 
I don't think using `*/${sourceBranch}` is substantially different

Reply:
Leaving a single version. It should be much easier to follow now.